website/pages/en/developing/assemblyscript-api.mdx

```diff
@@ -298,6 +298,33 @@ if (transfer == null) {
 
 > Note: If there is no entity created in the given block, `loadInBlock` will return `null` even if there is an entity with the given ID in the store.
 
+#### Looking up derived entities
+
+As of `graph-node` v0.31.0, `@graphprotocol/graph-ts` v0.31.0 and `@graphprotocol/graph-cli` v0.51.0 the `loadRelated` method is available.
+
+Given a derived entity type, yuo can load the entity field was derived from. For example, given the following schema:
```

Comment:
Might replace with:
This enables loading derived entity fields from within an event handler. For example, given the following schema: